tools/codegen/gen.py

```diff
@@ -276,9 +284,7 @@ def __call__(self, f: NativeFunction) -> Optional[str]:
                 defns += f"""
 // aten::{f.func}
 {sig.defn(name=method_name, is_redispatching_fn=is_redispatching_fn)} {{
-    static auto op = c10::Dispatcher::singleton()
-        .findSchemaOrThrow("aten::{f.func.name.name}", "{f.func.name.overload_name}")
-        .typed<{sig.type()}>();
+    static auto op = create_{name}_typed_handle();
```

Comment:
Regarding the change here, would it be possible to make use of the ATen Operators API to do the handle creation here? If the typed handle function doesn't exist on the class, you would add it to the class (rather than having a new top level function which you have to namespace, etc.)

Reply:
I kept it in the source file to avoid additional header includes in `Operators.h`. IMO this is fine because all you can do with a `TypedOperatorHandle` is `call` and `redispatch` so at that point you should just use the static functions, right?